src/components/AuthenticateModalContent.tsx

```diff
@@ -25,6 +25,13 @@ export const AuthenticateModalContent: FunctionComponent<Props> = ({ source }) =
             className="mt-4"
```

Comment:
If you're making more changes here, we could change `Gitlab` to `GitLab` on the label right ^^^ there